guidelines/content-guidelines/04-formatting.md

```diff
@@ -36,6 +36,42 @@ Metadata names            | When you create a Markdown document, define its `tit
 >**NOTE:** When you mention specific configuration files in your documents, consider linking to them instead of just mentioning their names. When you link to a file, use its name without the format extension. See the following example:
 > `To adjust the number of Pods in your Deployment, edit the [deployment](./deployment.yaml) file.`
 
+
+### Omission in code
+
+Often, when you quote snippets of code, it is not necessary to include them in full. For example, when you quote an HTTP response, most often, it is enough to only quote the relevant part, and there is no need to enclose the whole thing. In such a case, replace the omitted parts with `...` to signalize to the reader that something has been removed. 
```

Comment:
```suggestion
Often, when you quote snippets of code, it is not necessary to include them in full. For example, when you quote an HTTP response, it is enough to quote only the relevant part instead of enclosing the whole response. In such a case, replace the omitted parts with `...` to signalize that something has been removed. 
```